packages/tables/resources/views/columns/text-column.blade.php

```diff
@@ -189,8 +199,25 @@
             @if ($limitedArrayStateCount = count($limitedArrayState ?? []))
                 <{{ $isListWithLineBreaks ? 'li' : 'div' }}
                     class="text-sm text-gray-500 dark:text-gray-400"
+                    x-on:click.prevent="isLimited = ! isLimited"
                 >
-                    {{ trans_choice('filament-tables::table.columns.text.more_list_items', $limitedArrayStateCount) }}
+                    <x-filament::link
```

Comment:
I feel like adding an icon (e.g. `+` or `-`) would make it more obvious that the link is a link after a long list of text. Not too sure yet though. What do you think?

Reply:
I'll try it and see.

Reply:
The font weight is already enough IMO, as well as the underline on hover. It is pretty clear that it's not just another list item.